app/views/discovered_hosts/_discovered_host.html.erb

```diff
@@ -6,4 +6,7 @@
 <% unless defined? hide_disk %>
 <td class="hidden-tablet hidden-xs"><%= discovery_attribute(host, :disk_count) %></td>
 <td class="hidden-tablet hidden-xs"><%= number_to_human_size(discovery_attribute(host, :disks_size, 0)) %></td>
+<% if !Setting['discovery_fact_column'].nil? && Setting['discovery_fact_column'].size > 0 %>
```

Comment:
But `try` won't work for `nil` and also it does not compare to numbers. Isn't there a nicer way anyway?